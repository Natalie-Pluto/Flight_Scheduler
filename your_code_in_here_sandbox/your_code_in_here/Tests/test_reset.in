location import locations.csv
flight add wednesday 6:00 sydney perth 180
flight add wednesday 8:00 sydney perth 180
flight add wednesday 11:31 perth johannesburg 230
flights
flight
flight reset
flight -1 reset
flight 5 reset
flight 0 reset
flight 0
flight 1 reset
flight 1
exit