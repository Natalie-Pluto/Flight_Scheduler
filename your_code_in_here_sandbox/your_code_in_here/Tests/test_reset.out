User: Imported 51 locations.

User: Successfully added Flight 0.

User: Successfully added Flight 1.

User: Successfully added Flight 2.

User: Flights
-------------------------------------------------------
ID   Departure   Arrival     Source --> Destination
-------------------------------------------------------
   0 Wed 06:00   Wed 10:31   Sydney --> Perth
   1 Wed 08:00   Wed 12:31   Sydney --> Perth
   2 Wed 11:31   Wed 23:03   Perth --> Johannesburg

User: Usage:
FLIGHT <id> [BOOK/REMOVE/RESET] [num]
FLIGHT ADD <departure time> <from> <to> <capacity>
FLIGHT IMPORT/EXPORT <filename>

User:  Invalid Flight ID.

User: Invalid Flight ID.

User: Invalid Flight ID.

User: Reset passengers booked to 0 for Flight 0, Wed 06:00 Sydney --> Perth.

User: Flight 0
Departure:    Wed 06:00 Sydney
Arrival:      Wed 10:31 Perth
Distance:     3,254km
Duration:     4h 31m
Ticket Cost:  $989.16
Passengers:   0/180

User: Reset passengers booked to 0 for Flight 1, Wed 08:00 Sydney --> Perth.

User: Flight 1
Departure:    Wed 08:00 Sydney
Arrival:      Wed 12:31 Perth
Distance:     3,254km
Duration:     4h 31m
Ticket Cost:  $989.16
Passengers:   0/180

User: Application closed.

